src/model.js

```diff
@@ -0,0 +1,6 @@
+import falcor from '@graphistry/falcor';
```

Comment:
@mattiaerre ah, I was wondering if that was a factor. We have a redux app in production right now that's using [`@graphistry/falcor-react-redux`](https://github.com/graphistry/falcor/blob/master/packages/falcor-react-redux). As we built it out, we ended up writing everything async as redux-observable epics, since all the async Falcor methods return Observables.

In practice this means we're not using redux for anything beyond the global message bus, and lately it's made more sense to compose all that with recompose's `mapPropsStream`. As it stands, `falcor-react-redux` is more like a Falcor version of Relay, rather than true redux integration.

The good thing is we've built in compositional queries, and (with the new `recycleJSON` flag) super fast cache searches/react reconciliation. With this approach, Falcor is like an immutable asynchronous JSON store.